src/pages/ApplicationsPage.jsx

```diff
@@ -114,59 +122,63 @@ function ApplicationsPage(props) {
+                                                listItems={visibleApplications
+                                                    .filter((a) => a.applier.id === user.id)
+                                                    .map((a) => ({ id: a.id, title: a.protocol.title }))}
+                                                hsl={[36, 98, 83]}
+                                                allowEdit={true}
+                                                allowDelete={true}
+                                                viewFunction={(id) => navigate(`${id}`)}
+                                                editFunction={(id) => navigate(`${id}/manage`)}
+                                                deleteFunction={(id) => deleteApplication(id)}
+                                            />
+                                        </div>
+                                    )}
+                                    <div className="col-12 col-lg-6 d-flex flex-column m-vh-80 h-lg-100">
+                                        <h1 className="color-grey font-century-gothic text-nowrap fw-bold fs-3 mb-4">
+                                            Aplicações disponíveis
+                                        </h1>
                                         <ProtocolList
-                                            listItems={visibleApplications
-                                                .filter((a) => a.applier.id === user.id)
-                                                .map((a) => ({ id: a.id, title: a.protocol.title }))}
-                                            hsl={[36, 98, 83]}
-                                            allowEdit={true}
-                                            allowDelete={true}
+                                            listItems={visibleApplications.map((a) => ({ id: a.id, title: a.protocol.title }))}
+                                            hsl={[16, 100, 88]}
                                             viewFunction={(id) => navigate(`${id}`)}
-                                            editFunction={(id) => navigate(`${id}/manage`)}
-                                            deleteFunction={(id) => deleteApplication(id)}
                                         />
                                     </div>
                                 </div>
-                                <div className="col-12 col-md-10 col-lg-5 d-flex flex-column mh-100 h-lg-100 p-4 pt-lg-0">
-                                    <h1 className="color-grey font-century-gothic text-nowrap fw-bold fs-3 pb-4 m-0">
-                                        Aplicações disponíveis
-                                    </h1>
-                                    <div className="d-flex justify-content-center flex-grow-1 overflow-hidden">
-                                        <ProtocolList
+                            ) : (
+                                <div class="row flex-grow-1 overflow-y-hidden pb-lg-4 g-4">
```

Comment:
```suggestion
                                <div className="row flex-grow-1 overflow-lg-y-hidden pb-lg-4 g-4">
```

A ausência desse "lg" torna essa div inconsistente com o que seria a mesma div caso fosse dash.

Reply:
Não entendi;